tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -413,6 +413,9 @@ static llvm::cl::opt<bool> CodeCompleteCallPatternHeuristics(
         "Use heuristics to guess whether we want call pattern completions"),
     llvm::cl::cat(Category));
 
+// This option and the upcoming language feature ImportObjcForwardDeclarations
+// enable the same behavior. If we no longer rely upon
+// -enable-objc-forward-declarations, we may remove the option.
```

Comment:
IIUC swift-ide-test is supposed to be an "internal tool", so its options aren't guaranteed to be stable. I have no idea who uses this option in other projects, but if they are migrated, the flag can be deleted.

Reply:
I did not see this one in my compiled 5.9 `swift-ide-test`. Did I miss it or does it appear after that branch?

In any case, I 100% for removing this, if there's an alternate spelling which is closer to the compiler actualy spelling.

Reply:
It's there for me 🤷

Reply:
If tests pass with the flag removed, we can remove it. `swift-ide-test` is mainly intended for testing purposes.